src/main/java/calculator/view/InputView.java

```diff
@@ -0,0 +1,16 @@
+package calculator.view;
+import camp.nextstep.edu.missionutils.Console;
+public class InputView {
+    //TODO 입력 화면 구현
+
+    //입력과 화면 출력을 구분
+    public void printInputView() {
+        System.out.println("덧셈할 문자열을 입력해 주세요.");
+    }
+
+    public String getInput() {
+        return Console.readLine();
+    }
```

Comment:
이 부분을 합쳐서

```suggestion
    public void getInput(){
    }
    public void getInput(msg){
    }
}
```
이런식으로 오버로드해서 해도 괜찮을거 같아요. printInputView를 분리하는게 이점이 얼마나 있을지 조금 의문점이 있습니다!